apps/remote/src/app/remote-module/remote-module.component.css

```diff
@@ -0,0 +1,6 @@
+:host {
+  display: block;
+  background-color: beige;
```

Comment:
do all these text colors look good? should not we use some colors by Taiga better https://taiga-ui.dev/colors ?